src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.Sdk.BeforeCommon.targets

```diff
@@ -92,6 +92,24 @@ Copyright (c) .NET Foundation. All rights reserved.
     <Reference Include="System.Xml"/>
   </ItemGroup>
 
+  <!-- Set the default versions of the NETStandard.Library or Microsoft.NETCore.App packages to reference.
+       The implicit package references themselves are defined in Microsoft.NET.Sdk.props, so that they can be overridden
+       in the project file. -->
+  <PropertyGroup Condition="'$(TargetFrameworkIdentifier)' == '.NETStandard' And '$(NetStandardDefaultPackageVersion)' == ''">
+    <NetStandardDefaultPackageVersion>1.6.0</NetStandardDefaultPackageVersion>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(TargetFrameworkIdentifier)' == '.NETCoreApp' And '$(NetCoreAppDefaultPackageVersion)' == ''" >
+    
+    <!-- For Microsoft.NETCore.App, default to using the version of the package that matches the target framework version -->
+    <NetCoreAppDefaultPackageVersion>$(_TargetFrameworkVersionWithoutV)</NetCoreAppDefaultPackageVersion>
+    
+    <!-- When the TFM is .NET Core 1.0, use version 1.0.1 of the Microsoft.NETCore.App package -->
+    <NetCoreAppDefaultPackageVersion Condition=" '$(_TargetFrameworkVersionWithoutV)' == '1.0' ">1.0.1</NetCoreAppDefaultPackageVersion>
```

Comment:
Naming question: How is "default" being different from "implicit"? Currently, there is a concept of "implicit" framework references. But then there is a "default" version?
My intuition would suggest there is a version property and if not set, a default would be used. Here, only the "default" is used if I'm not mistaken.
I feel it would be clearer if the property is named `ImplicitNetCoreAppPackageVersion`. Or simply `NetCoreAppPackageVersion`. Does that make sense?